### DISCUSSION TITLE
cancellation and refund

### DISCUSSION THREAD

**Discussion by imbrups** [aclap-dev/video-downloadhelper#1038] [License & Purchase requests]
### What's your issue?

Hi, 

Last week I bought the license to do some projects that were planned for the next few months, but they were cancelled, so I won't need to use the tool anymore.

As it will be a very high cost for me, I would like a cancellation and refund, please

Thanks

### The 10 first digits (only 10) of your license number or email address

brunapsnt@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
